clearer instructions when using workers subdomain (#2120)
cloudflare/cloudflare-docs: products/workers/src/content/platform/sites/start-from-scratch.md

  $ wrangler dev
  ```

5. Decide if you would like to publish your site to a [`*.workers.dev` subdomain](/get-started/guide#configure-for-deploying-to-workersdev) or a [custom domain](/get-started/guide#optional-configure-for-deploying-to-a-registered-domain) that you own and have already attached as a Cloudflare zone. Then update your `wrangler.toml` file accordingly:

  **`*.workers.dev`**: Enable the `workers_dev` configuration.

  ```toml
  workers_dev = true
  ```
  
  And/Or

  **Personal Domain**: Add your `zone_id` and a `route`.

  ```toml
  zone_id = "42ef.."
  route = "https://example.com/*"
  ```

>   <Aside>

Reviewer: ```suggestion
  <Aside type="note">
```

>   __Note:__ Check out documentation on [Routes](/platform/routes) to configure `route` properly.

Reviewer: ```suggestion
Refer to the documentation on [Routes](/platform/routes) to configure `route` properly.
```

  $ wrangler dev
  ```

5. Decide if you would like to publish your site to a [`*.workers.dev` subdomain](/get-started/guide#configure-for-deploying-to-workersdev) or a [custom domain](/get-started/guide#optional-configure-for-deploying-to-a-registered-domain) that you own and have already attached as a Cloudflare zone. Then update your `wrangler.toml` file accordingly:

  **`*.workers.dev`**: Enable the `workers_dev` configuration.

  ```toml
  workers_dev = true
  ```
  
  And/Or

  **Personal Domain**: Add your `zone_id` and a `route`.

  ```toml
  zone_id = "42ef.."
  route = "https://example.com/*"
  ```

  <Aside type="note">

>     Refer to the documentation on [Routes](/platform/routes) to configure `route` properly.

Reviewer: ```suggestion
Refer to the documentation on [Routes](/platform/routes) to configure `route` properly.
```

  $ wrangler preview --watch
  ```

> 5. Decide where you’d like to publish your site to: [a workers.dev subdomain](/get-started/guide#configure-for-deploying-to-workersdev) or your [personal domain](/get-started/guide#optional-configure-for-deploying-to-a-registered-domain) registered with Cloudflare. Update your `wrangler.toml`:

Reviewer: ```suggestion
5. Choose whether you would like to publish your site to: [a `*.workers.dev` subdomain](/get-started/guide#configure-for-deploying-to-workersdev) or your [personal domain](/get-started/guide#optional-configure-for-deploying-to-a-registered-domain) registered with Cloudflare. Then, update your `wrangler.toml` file accordingly:
```

>   **workers.dev**: Make sure `workers_dev` is set to true. This is the default.

Reviewer: ```suggestion
  **`*.workers.dev`**: Make sure `workers_dev` is set to true. This is the default.
```

> Configure route to match your workers.dev subdomain: 
>   ```toml
>   route = "your-workers-subdomain.workers.dev/*"
>   ```
> 

Reviewer: ```suggestion
Configure `route` to match your `*.workers.dev` subdomain: 

  ```toml
  route = "your-workers-subdomain.workers.dev/*"
  ```

```

  $ wrangler preview --watch
  ```

> 5. Choose whether you would like to publish your site to: [a `*.workers.dev` subdomain](/get-started/guide#configure-for-deploying-to-workersdev) or your [personal domain](/get-started/guide#optional-configure-for-deploying-to-a-registered-domain) registered with Cloudflare. Then, update your `wrangler.toml` file accordingly:

Reviewer 2: While this _could be_ said for the original text, the new text makes it sound like you can **only** pick one or the other.

When `workers_dev` is true, then it deploys to the subdomain & all routes are assumed. I'm fairly certain it cannot be modified and/or narrowed down to explicit paths on the subdomain.

You must define a `zone_id` and `route` (or `routes`) combination in order to deploy to a custom zone.

>   **`*.workers.dev`**: Make sure `workers_dev` is set to true. This is the default.

Reviewer 2: It is not on by default: https://developers.cloudflare.com/workers/cli-wrangler/configuration#keys

```suggestion
  **`*.workers.dev`**: Enable the `workers_dev` configuration.
```

Configure `route` to match your `*.workers.dev` subdomain: 

  ```toml

>   route = "your-workers-subdomain.workers.dev/*"

Reviewer 2: ```suggestion
  workers_dev = true
```

  ```
  

>   Or

Reviewer 2: lol – what to do here?

```suggestion
  And/Or
```
